src/index.ts

```diff
@@ -76,9 +82,10 @@ ast.getSourceFiles().forEach(sourceFile => {
     }
 
     writeFileSync(outputPath.replace(/\.ts$/, mockClassFilenameSuffix), mockedSource);
```

Comment:
Console log output paths only. Remove any other logs in the program (except errors).